### PR TITLE
fix: README.md & dev install details (rb version, pgsql & environment, redis)

### PR DESCRIPTION
**Pull request by JeremiGendron**
image

Version in README.md is incorrect
image

```
No pg_config... trying anyway. If building fails, please try again with
 --with-pg-config=/path/to/pg_config
checking for libpq-fe.h... no
Can't find the 'libpq-fe.h header
```

The above is fixed by installing libpq-dev on Ubuntu 18.04.

### CODE REVIEW

**Comment by JeremiGendron** [comment]
closes #535 

Changes made:

 - rb version was wrong in README.md
 - no details as to postgresql setup
 - no mention of redis
 - janky password environment variable (everything uses PG_*, but password uses PGPASSWORD - added a commented field to .env-example to avoid breaking ci)

Everything is documented in commit messages and the README.md

Any instructions welcome, obviously not a regular here :)

**Comment by coderberry** [comment]
@hopsoft Please check out this PR when you get a moment.

@JeremiGendron Thank you!!!!!

File: README.md [hunk 1]
```diff
@@ -140,8 +140,13 @@ ENUMS::USER_ROLES.constants
...
 
-- ruby version `2.6.2` via [rbenv](https://github.com/rbenv/rbenv)
-- `brew install graphviz`
+- ruby version `2.6.3` via [rbenv](https://github.com/rbenv/rbenv)
+- NodeJS version `<12.0.0` via [nvm](https://github.com/nvm-sh/nvm) with yarn installed globally
+- graphviz `brew install graphviz` or `sudo apt-get install graphviz`
+- PostgreSQL 11
+- Redis
+
+You must create a (superuser) role with the name of your OS user in your postgres configuration in order to run db operations (e.g. testing and development).
```
Comment (mglaman) [inline README.md]: I was able to get up and running with `brew install postgre` and then use the following without running it as a full service.

Start: `pg_ctl -D /usr/local/var/postgres start`
Stop:`pg_ctl -D /usr/local/var/postgres stop`

Comment (mglaman) [reply README.md]: (no configuration required)

**Comment by JeremiGendron** [comment]
ok, on Ubuntu this was definitely a required step.

**Comment by mglaman** [comment]
@JeremiGendron 👍 I just wanted to mention, as I had set up locally this week

**Comment by coderberry** [comment]
Great job!

**Pull request merged**